### ISSUE TITLE
Add a website

### ISSUE THREAD

**Issue by deadofgamers1999** [elgatito/script.elementum.burst#377]
Hi!
I would like someone to add https://newinsane.info 
(Insane Tracker)

**Comment by antonsoroko** [comment]
JFYI: looks like login on this website is protected by adding randomly generated numbers into POST request, so it will not work unless someone will add a "workaround" for such case to burst code (python code).

**Comment by antonsoroko** [comment]
@elgatito если разбираешься в вебе - можешь пожалуйста посмотреть как тут организована защита от автологина? они добавляют 2 переменные `x` и `y` с значениямя случайных чисел, но где они это делают я не нашёл, хотя искал вроде по всех их javascript файлам. как минимум интересно как они это делают, ну и ещё можем ли мы это обойти. спасибо.

**Comment by elgatito** [comment]
@antonsoroko Да, по идее должен быть js, но я его вообще не вижу там и браузер тоже не видит, а как-то добавляет параметры...
